scripts/docksal.bat

```diff
@@ -0,0 +1,27 @@
+@echo off
```

Comment:
I don't we need the entire `babun.bat` script here. This script can be as simple as:

```
@echo off
setlocal enableextensions enabledelayedexpansion

set SCRIPT_PATH=%~dp0
set SCRIPT_PATH=%SCRIPT_PATH:\=/%
set BABUN_HOME=%SCRIPT_PATH%
set CHERE_INVOKING=1
set CYGWIN_HOME=%BABUN_HOME%\cygwin

start "" "%CYGWIN_HOME%\bin\bash.exe" --login

```

Notice I switched from zsh.exe to bash.exe. zsh.exe should not be invoked directly with Babun as that results is some settings not being applied - http://take.ms/ZD6Sdr